geniza/corpus/views.py

```diff
@@ -99,6 +100,9 @@ def page_title(self):
     def page_description(self):
         return Truncator(self.get_object().description).words(20)
 
+    def permalink(self):
+        return absolutize_url(self.get_object().get_absolute_url())
```

Comment:
I love that you made this a method — but I'd like to have it on the `Document` object. It will eventually be useful in other contexts also, like the data exports. And that way we won't have to pass anything else in the view context, just reference `document.permalink` in the template